pkg/execution/state/redis_state/queue_processor.go

```diff
@@ -523,25 +516,32 @@ func (q *queue) scan(ctx context.Context) error {
 			return nil
 		}
 
-		eg := errgroup.Group{}
-
 		// Reduce number of peeked partitions as we're processing multiple accounts in parallel
 		// Note: This is not optimal as some accounts may have fewer partitions than others and
 		// we're leaving capacity on the table. We'll need to find a better way to determine the
 		// optimal peek size in this case.
-		peekSize := int64(math.Round(float64(PartitionPeekMax / int64(len(peekedAccounts)))))
+		// accountPartitionPeekMax := int64(math.Round(float64(PartitionPeekMax / int64(len(peekedAccounts)))))
+		accountPartitionPeekMax := int64(10)
```

Comment:
This is the result of manual tuning, balancing hitting QPS limits (running into context canceled and context deadline exceeded) and processing workloads.